nautobot/docs/development/apps/api/platform-features/uniquely-identify-objects.md

```diff
@@ -0,0 +1,60 @@
+# Uniquely Identifying a Nautobot Object
```

Comment:
I don't know where to put it but should we mention that `GitRepository` still has its `slug` field?

Reply:
I think that's out of scope for this document. It belongs in the 2.0 upgrade guide